benchmarks/objectExtractionRunningTime.py

```diff
@@ -44,25 +59,39 @@
 
 logger = logging.getLogger(__name__)
 
+# Define the set of features to extract
 NAME = "Standard Object Features"
 
 FEATURES = {
     NAME: {
-        "Count": {},
-        "RegionCenter": {},
-        "Coord<Principal<Kurtosis>>": {},
-        "Coord<Minimum>": {},
-        "Coord<Maximum>": {},
+        "Count": {},                       # Total count of pixels in object
+        "RegionCenter": {},                # Center of mass
+        "Coord<Principal<Kurtosis>>": {},  # Measure of shape "peakedness"
+        "Coord<Minimum>": {},              # Minimum coordinates (bounding box)
+        "Coord<Maximum>": {},              # Maximum coordinates (bounding box)
     }
 }
 
-# Cleanup functions (used in vigra_objfeats.py)
```

Comment:
Why remove the pointer to vigra_objfeats.py? These kind of pointers make it easy to identify that this is code duplicated from another file - although the phrasing could be improved to make that clearer.